lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -97,16 +80,22 @@
 '''
 
 EXAMPLES = '''
-# Basic server certificate upload
-tasks:
-- name: Upload Certificate
-  iam_cert:
+# Basic server certificate upload from local file
+- iam_cert:
     name: very_ssl
     state: present
-    cert: somecert.pem
-    key: privcertkey
-    cert_chain: myverytrustedchain
+    cert: "{{ lookup('file', 'path/to/cert') }}"
```

Comment:
I know this has already been merged, however did this make the module less friendly for those using it on remote hosts?  i.e.  lookup will not work and therefore slurp would have to be used, and an extra task added in order to achieve the same thing.